net.sourceforge.pdfchain.json

```diff
@@ -0,0 +1,84 @@
+{
+    "app-id": "net.sourceforge.pdfchain",
+    "runtime": "org.gnome.Platform",
```

Comment:
do you really need the GNOME SDK?

Reply:
That's not the SDK, it's the runtime.. or did you mean other thing?

Reply:
potato / potato.
Isn't freedekstop-sdk enough? It has gtk3.

Reply:
Yeah I just tried and it worked fine. I'm about to submit changes in a moment.